Update usage instructions of prep_for_subm.sh (#319)
submariner-io/submariner-website: src/content/quickstart/openshift/ready_clusters.md

### Make your clusters ready for Submariner

Submariner gateway nodes need to be able to accept traffic over UDP ports (4500 and 500 by default) when using IPSEC.
In addition we use port 4800/UDP to encapsulate traffic from the worker nodes to the gateway nodes and ensuring that Pod IP addresses are preserved.

Additionally, the default OpenShift deployments don't allow assigning an elastic public IP
to existing worker nodes, something that it's necessary at least on one end of the IPSEC connections.

To handle all those details we provide a script
[prep_for_subm.sh](https://github.com/submariner-io/submariner/blob/master/tools/openshift/ocp-ipi-aws/prep_for_subm.sh)
that will prepare your AWS OpenShift deployment for Submariner, and will create an additional gateway node with an external IP.

> See instructions how to [make-your-clusters-ready-for-submariner](https://submariner.io/quickstart/openshift/ready_clusters/#make-your-clusters-ready-for-submariner).

Reviewer: IMO, we can keep the bash commands here instead of a separate page as its hardly three or four steps.

Author: Duplicating instructions will require duplicating maintenance on both pages. @nyechiel what do you think ?

Reviewer: I'm not saying to duplicate the instructions. Please take a look at the rendered page, it gives some details but does not give the instructions. 

![Screenshot from 2020-10-19 20-41-00](https://user-images.githubusercontent.com/6574656/96470136-96fde180-124b-11eb-9215-6fd5b4c2a407.png)

Now, if I click on the hyperlink, it repeats the same instructions (along with bash commands) and the rendering on the left pane shows a white line. So, my suggestion is instead of showing the bash instructions in a separate page, lets include them here itself.

![Screenshot from 2020-10-19 20-43-08](https://user-images.githubusercontent.com/6574656/96470294-c3b1f900-124b-11eb-9053-2c5552f8dc48.png)

### Make your clusters ready for Submariner

> Submariner gateway nodes need to be able to accept traffic over UDP ports (4500 and 500 by default) when using IPSEC.

Reviewer 2: ```suggestion
Submariner gateway nodes need to be able to accept traffic over UDP ports (4500 and 500 by default) when using IPsec.
```

In addition we use port 4800/UDP to encapsulate traffic from the worker nodes to the gateway nodes and ensuring that Pod IP addresses are preserved.

> Additionally, the default OpenShift deployments don't allow assigning an elastic public IP

Reviewer 2: ```suggestion
Additionally, the default OpenShift deployment does not allow assigning an elastic public IP
```

> to existing worker nodes, something that it's necessary at least on one end of the IPSEC connections.

Reviewer 2: ```suggestion
to existing worker nodes, something that is necessary at least on one end of the IPsec connections.
```

To handle all those details we provide a script
[prep_for_subm.sh](https://github.com/submariner-io/submariner/blob/master/tools/openshift/ocp-ipi-aws/prep_for_subm.sh)
that will prepare your AWS OpenShift deployment for Submariner, and will create an additional gateway node with an external IP.

In the following example, we create the gateway node on cluster-b, with custom IPsec ports and instance type:

```bash

curl https://raw.githubusercontent.com/submariner-io/submariner/master/tools/openshift/ocp-ipi-aws/prep_for_subm.sh -L -O
chmod a+x ./prep_for_subm.sh

export IPSEC_NATT_PORT=4501
export IPSEC_IKE_PORT=501
export GW_INSTANCE_TYPE=m4.xlarge

> 

Reviewer 3: Can we create an additional section with those parameters, and one without?, those parameters are optional otherwise people will copy paste all of it (including the env vars).

See the example here:  https://github.com/submariner-io/submariner/tree/master/tools/openshift/ocp-ipi-aws

Author: done

### Make your clusters ready for Submariner

Submariner gateway nodes need to be able to accept traffic over UDP ports (4500 and 500 by default) when using IPSEC.
In addition we use port 4800/UDP to encapsulate traffic from the worker nodes to the gateway nodes and ensuring that Pod IP addresses are preserved.

Additionally, the default OpenShift deployment does not allow assigning an elastic public IP

> to existing worker nodes, something that it is necessary at least on one end of the IPsec connections.

Reviewer 2: ```suggestion
to existing worker nodes, something that is necessary at least on one end of the IPsec connections.
```
